src/schemas/category.schema.ts

```diff
@@ -0,0 +1,11 @@
+import { object, string, ref } from 'yup';
+
+export const createCategorySchema = object({
+    body: object({
+        label: string()
+            .required(`Label is required`)
+            .min(2, `The label is to short - should be 2 chars minium`)
+            .max(75, `The label is to long - should be 75 chars maximum`)
+            .matches(/^[a-zA-Z0-9_.-]*$/, `Last name can only contain Latin letters.`),
```

Comment:
Maybe you can change this reggex by an another reggex who allow only letters and not numbers.